tests/lua/thread.lua

```diff
@@ -1,5 +1,7 @@
 require "bootstrap-tests"
 
+local effil = effil
```

Comment:
What is it for?

Reply:
here we make `effil` be local variable, so we can capture it in upvalue